OsModule.py

```diff
@@ -0,0 +1,18 @@
+import os 
+print(os.name)
+#"os.name" function gives the name of the operating system dependent module imported.
+
+fd = "test.txt"
+  
+# popen() is similar to open() 
+file = open(fd, 'w') 
+file.write("Hello") 
+file.close() 
+file = open(fd, 'r') 
+text = file.read() 
+print(text) 
+  
+# popen() provides a pipe/gateway and accesses the file directly 
```

Comment:
Great Work !! Arjun,
From next time try to add comments without leaving blank spaces.